packages/integration-tests/tests/__snapshots__/v1014-multi-project-manual-config.test.ts.snap

```diff
@@ -12,42 +12,63 @@ __ROOT__/v1014-multi-project-manual-config/src/app/app.component.spec.ts
   12:3   error  Use @Input rather than the \`inputs\` metadata property (https://angular.io/styleguide#style-05-12)                       @angular-eslint/no-inputs-metadata-property
   13:3   error  Use @Output rather than the \`outputs\` metadata property (https://angular.io/styleguide#style-05-12)                     @angular-eslint/no-outputs-metadata-property
   14:3   error  Use @HostBinding or @HostListener rather than the \`host\` metadata property (https://angular.io/styleguide#style-06-03)  @angular-eslint/no-host-metadata-property
+  16:14  error  Pipes should implement PipeTransform interface                                                                          @angular-eslint/use-pipe-transform-interface
   18:3   error  In the class \\"ExampleComponent\\", the output property \\"onFoo\\" should not be prefixed with on                             @angular-eslint/no-output-on-prefix
    6:35  error  Invalid binding syntax. Use [(expr)] instead                                                                            @angular-eslint/template/banana-in-box
    8:15  error  Invalid binding syntax. Use [(expr)] instead                                                                            @angular-eslint/template/banana-in-box
    8:29  error  Invalid binding syntax. Use [(expr)] instead                                                                            @angular-eslint/template/banana-in-box
    9:48  error  Invalid binding syntax. Use [(expr)] instead                                                                            @angular-eslint/template/banana-in-box
 
-✖ 9 problems (9 errors, 0 warnings)
-  5 errors and 0 warnings potentially fixable with the \`--fix\` option.
+✖ 10 problems (10 errors, 0 warnings)
+  6 errors and 0 warnings potentially fixable with the \`--fix\` option.
 
 
 __ROOT__/v1014-multi-project-manual-config/src/app/example.pipe.ts
-  6:8  error  Classes decorated with @Pipe decorator should implement PipeTransform interface  @angular-eslint/use-pipe-transform-interface
+  6:14  error  Pipes should implement PipeTransform interface  @angular-eslint/use-pipe-transform-interface
 
 ✖ 1 problem (1 error, 0 warnings)
+  1 error and 0 warnings potentially fixable with the \`--fix\` option.
 
 
 __ROOT__/v1014-multi-project-manual-config/src/app/multiple-inline-templates.page.ts
-   4:13  error  Strings must use singlequote                  @typescript-eslint/quotes
-  12:13  error  Strings must use singlequote                  @typescript-eslint/quotes
-   6:35  error  Invalid binding syntax. Use [(expr)] instead  @angular-eslint/template/banana-in-box
-  14:48  error  Invalid binding syntax. Use [(expr)] instead  @angular-eslint/template/banana-in-box
+   4:13  error  Strings must use singlequote                    @typescript-eslint/quotes
+   9:14  error  Pipes should implement PipeTransform interface  @angular-eslint/use-pipe-transform-interface
```

Comment:
Hmm, please make sure you actually verify all these changes, it is a red flag that adding a fixer would cause so many changes to the integration-tests and I think most of them are regressions.

The source of this file I have commented on is:

```ts
import { Component } from '@angular/core';

@Component({
  selector: "app-one-inline-template",
  template: `
    <input type="text" name="foo" ([ngModel])="foo">
  `,
})
export class OneInlineTemplateComponent {}

@Component({
  selector: "app-two-inline-template",
  template: `
    <div [oneWay]="oneWay" (emitter)="emitter" ([twoWay])="twoWay"></div>
  `,
})
export class TwoInlineTemplateComponent {}
```

The `use-pipe-transform-interface` rule should not be erroring for that source, right?

I believe this is the case for a number of these changes

Reply:
Ups, it should be fine now.